client/components/.eslintrc

```diff
@@ -0,0 +1,15 @@
+{
+  "env": {
+    "browser": true,
+    "node": true
+  },
+  "settings": {
+    "ecmascript": 6,
+    "jsx": true
+  },
+  "parser": "babel-eslint",
+  "rules": {
+    "quotes": [1, "single", "avoid-escape"],
+    "no-unused-vars": 0
+  }
+}
```

Comment:
For react components a useful eslint plugin to add is: https://github.com/yannickcr/eslint-plugin-react. Not required for this PR, but would be nice to add.